scss/_topslider.scss

```diff
@@ -56,19 +62,68 @@
     }
   }
 }
-.diagonal-box {
-  position: relative;
-  transform: skewY(0.5deg);
-  height: 100px;
-  top: -80px;
-  z-index: 1;
-  background-color: transparentize($color: white, $amount: 0.5);
+
+@media screen and (max-width: 1300px) {
+  .headSwiper {
+    .swiper-text {
+      width: 350px;
+    }
+    .swiper-text-h1 {
+      font-size: 45px;
+    }
+  }
+}
+
+@media screen and (max-width: 1000px) {
+  .headSwiper {
+    .swiper-text {
+      width: 300px;
+    }
+    .swiper-text-h1 {
+      font-size: 40px;
+    }
+    .swiper-text-p {
+      font-size: 14px;
+      line-height: 25px;
+    }
+  }
+}
+
+@media screen and (max-width: 900px) {
```

Comment:
Uglavnom bi trebao da imas samo 3 breakpointa, i to:
`767px` - mobile
`1024` - tablet
`1200px` - desktop

Mozes napraviti te variable i mixine, pa da ih rejuzujes, da ne kazem: iskoristavas :)

Reply:
![image](https://user-images.githubusercontent.com/38419355/142232653-245b00b2-bed2-4da0-a8b0-4b35ab8c0ead.png)
![image](https://user-images.githubusercontent.com/38419355/142232784-092a765f-9279-4b5e-8fcb-317f7be2199b.png)
Ovako nekako